packages/shared/src/api/dto/user/update.user.dto.ts

```diff
@@ -0,0 +1,6 @@
+export interface UpdateUserDto {
```

Comment:
move to
/home/michal/dev/priv/camp-trello-clone/packages/shared/src/api/command

and call: CreateUserCommand

we use dtos and commands to model our API